hostinit/setup.go

```diff
@@ -125,15 +139,25 @@ func (init *HostInit) startHosts(ctx context.Context) error {
 		})
```

Comment:
Super nit, but wouldn't "FQDN" or "hostname" be more appropriate?

Reply:
not sure, it's not reliable based on host provider. 

I think this is outside of the scope of this CR